src/Post/Post.css

```diff
@@ -1,45 +1,135 @@
 .post-content {
-  max-width: 614px;
   margin: 20px auto;
-  border: 1px solid #dbdbdb;
-  border-radius: 3px;
-  background: white;
+  background: #fff;
   display: grid;
   grid-template-areas:
     'header'
     'media'
+    'progress'
     'actions'
     'likes'
     'caption'
     'comments'
     'time'
     'input';
+  max-width: 100%;
+  border: 1px solid #efefef;
+  border-radius: 3px;
 }
-.post-content.is-expanded {
-  margin-top: 81px;
-  grid-template-columns: 1fr 1fr;
+.post-content.is-expanded:not(.is-floating) {
+  grid-template-columns: minmax(345px, max-content) minmax(200px, 335px);
```

Comment:
We need a border in this case. Visit http://localhost:3000/p/1 to see.

Reply:
Yea, On it